scripts/ci/jobs/eks_qa_e2e_tests.py

```
@@ -3,15 +3,155 @@
+    Scanner V4 DB requires PVCs, but EKS clusters don't have the EBS CSI driver
+    by default. The gp2 StorageClass exists but its provisioner (kubernetes.io/aws-ebs)
+    is redirected to ebs.csi.aws.com via CSI migration, which isn't running without
+    this addon.
+    """
+    print("Installing AWS EBS CSI driver addon...")
+
+    # Get cluster name from kubectl context or CLUSTER_NAME env var
+    cluster_name = os.environ.get("CLUSTER_NAME")
+    if not cluster_name:
+        result = subprocess.run(
+            ["kubectl", "config", "current-context"],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)